storage-market.md

```diff
@@ -44,6 +44,14 @@ type StorageMarket interface {
 	// accounting of the total storage in the storage market.
 	UpdateStorage(delta BytesAmount)
 
+  // PowerLookup returns a given miner's committed storage in the system.
+  // This number is used during leader election and block validation.
+  PowerLookup(miner Address) BytesAmount
+  
+  // Convenience method that simply returns whether a given miner's committed
+  // power is > 0.
+  IsMiner(miner Address) bool
```

Comment:
The state value for a miner's power is currently on the miner actor, not here. It would be more straightforward to consult the miner directly, and consult the market only for the boolean value of whether a miner is participating. This method will just have to send a message to the miner actor to work it out.

I don't think a "convenience" method that composes both belongs in the spec'd interfaces. We should require only the bare primitives.

(Alternatively, it would also make sense for the power values to be stored in the market actor instead).